src/components/ProjectDescription/index.jsx

```diff
@@ -12,12 +12,12 @@ export default function ProjectDescription() {
         </Col>
         <Col md={6}>
           <div className="projectDescription">
-            <h1>
+            <h2 className="projectDescriptionHeader">
```

Comment:
If you worked on projectDescription on the other PR can you remove it from here to avoid conflicts

Reply:
PTAL